sdk/core/core-tracing/src/instrumenter-state-cjs.cts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @internal
+ *
+ * The tracing instrumenter that can be replaced at runtime by OpenTelemetry instrumentation.
+ *
+ * Unknown is used to avoid importing `interfaces` from the CJS module, which would require dynamic imports.
```

Comment:
could we avoid that with a `type only` import? or would TS still complain about the module format

Reply:
Ah lovely, I think it works. Good callout

Reply:
Update, unfortunately it still was unhappy... 😢 

I think this PR, in order to fix tracing and get us into a healthy state, is ok to go in. Then, I can spend more time thinking about how I can structure this correctly. Is that ok with you? I'll make a follow-up issue